pallets/loans/rpc/runtime-api/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+use codec::Codec;
+use sp_runtime::{DispatchError, FixedU128};
+
+sp_api::decl_runtime_apis! {
+    pub trait LoansApi<AccountId> where
+        AccountId: Codec, {
+        fn get_account_liquidity(account: AccountId) -> Result<(FixedU128, FixedU128), DispatchError>;
```

Comment:
Type alias for FixedU128 should be more readable, like 
`type Shortfalls = FixedU128`
`type Surplus = FixedU128`

Reply:
Do we need an RPC to get all the shortfall accounts?

Reply:
This PR covers only `get_account_liquidity`